rollout/ephemeralmetadata_test.go

```diff
@@ -169,7 +169,7 @@ func TestSyncBlueGreenEphemeralMetadataSecondRevision(t *testing.T) {
 	f := newFixture(t)
 	defer f.Close()
 
-	r1 := newBlueGreenRollout("foo", 1, nil, "active", "preview")
+	r1 := newBlueGreenRollout("foo", 3, nil, "active", "preview")
```

Comment:
The number of replicas defined in the rollout were not equal to number in the replicasets. My change correctly caused an extra scaling event where reconcile the fact the replicas were out of sync.